backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -94,11 +107,19 @@ public String addPatient(
 			residentCongregateSetting,
 			employedInHealthcare
 		);
+
+		if (newPatient.getRole() != PersonRole.STAFF) {
+			Organization org = _os.getCurrentOrganization();
+			Facility facility = _facilityRepo.findByOrganizationAndInternalId(org, facilityId).orElseThrow();
+			newPatient.setFacility(facility);
+		}
+
 		_repo.save(newPatient);
 		return newPatient.getInternalId().toString();
```

Comment:
Since the mutator doesn't actually return this value, I know nobody is relying on it—can we just change it into `return _repo.save(newPatient);`? (And update the schema etc?)

Reply:
updated